core/domain/exp_services.py

```diff
@@ -1684,15 +1683,14 @@ def get_scaled_average_rating(ratings):
         return 0
     average_rating = get_average_rating(ratings)
     z = 1.9599639715843482
-    x = python_utils.divide((average_rating - 1), 4)
+    x = ((average_rating - 1) // 4)
```

Comment:
`average_rating` will most probably be `float`, should we use `/`?

Reply:
> `average_rating` will most probably be `float`, should we use `/`?

I have thought of these functions should return a float value. Next time I'll keep an eye on function too....